CustomMission/TRGMSetEnemyFaction.sqf

```diff
@@ -1,65 +0,0 @@
-
```

Comment:
This file is needed, see here: https://youtu.be/K6u34Ekxsmo?t=115
(have set the video to start at 1:55, as that is where i go into detail about the values)